pkg/login/social/connectors/github_oauth.go

```diff
@@ -45,6 +45,9 @@ type GithubTeam struct {
 	Organization struct {
 		Login string `json:"login"`
 	} `json:"organization"`
+	Parent struct {
```

Comment:
The goal is to match the APIs contract and `Parent` can be null.
```suggestion
	Parent *struct {
```